bridges/snowbridge/primitives/core/src/location.rs

```diff
@@ -230,4 +249,132 @@ mod tests {
 			);
 		}
 	}
+
+	#[test]
+	fn test_agent_id_fully_compatible() {
```

Comment:
What about tests for token ids? When PNA goes live in a few weeks, we are going to register DOT.

Will `TokenIdOf(DOT)` be compatible with `LocationHashOf(DOT)`. The Gateway contract does require that as it does lookup the ERC20 contract address based on the token id.

Reply:
Test for tokens ids.
https://github.com/yrong/polkadot-sdk/blob/c5bf3d2af59db7cc9f843979ce6aa6a74417a17a/bridges/snowbridge/primitives/core/src/location.rs#L132

Assert here.
https://github.com/yrong/polkadot-sdk/blob/c5bf3d2af59db7cc9f843979ce6aa6a74417a17a/bridges/snowbridge/primitives/core/src/location.rs#L228-L235